app/javascript/vue/label_template/insert_field_dropdown.vue

```diff
@@ -22,7 +22,7 @@
             :data-template="tooltipTemplate"
             class="field-element"
             :title="i18n.t('label_templates.show.insert_dropdown.field_code', {code: field.tag})"
-            @click="$emit('insertField', field.tag)"
+            @click="insertTag(field)"
```

Comment:
Naming consistency: it used to be called 'insertField' and emmited field.tag, not it's called insertTag, but takes field as an argument. Probably just call it insertField, and always use field as argument.